test/check-application

```diff
@@ -2051,6 +2052,21 @@ class TestFiles(testlib.MachineCase):
         def dir_file_count(directory: str) -> int:
             return int(m.execute(f"find {directory} -mindepth 1 -maxdepth 1 | wc -l").strip())
 
+        def assert_upload_alert(files: List[str], owner_group: Optional[str] = None, *, close: bool = True) -> None:
+            if len(files) > 1:
+                b.wait_in_text(".pf-v5-c-alert__title", "Files uploaded")
+                b.wait_in_text(".pf-v5-c-alert__description", f"{len(files)} files")
+            else:
+                b.wait_in_text(".pf-v5-c-alert__title", "File uploaded")
+                b.wait_in_text(".pf-v5-c-alert__description", files[0])
```

Comment:
You can probably be stricter here with `wait_text`?

Reply:
Yes, then concatenate them together.